iexec-blockchain-adapter-api-library/src/main/java/com/iexec/blockchain/api/BlockchainAdapterApiClientBuilder.java

```diff
@@ -2,21 +2,36 @@
 
 import com.iexec.common.utils.FeignBuilder;
 import feign.Logger;
-import feign.auth.BasicAuthRequestInterceptor;
 
+/**
+ * Creates Feign client instances to query REST endpoints described in {@link BlockchainAdapterApiClient}.
+ */
 public class BlockchainAdapterApiClientBuilder {
 
     private BlockchainAdapterApiClientBuilder() {}
 
-    public static BlockchainAdapterApiClient getInstance(Logger.Level logLevel,
-                                                         String url, String username, String password) {
-        BasicAuthRequestInterceptor requestInterceptor =
-                new BasicAuthRequestInterceptor(
-                        username,
-                        password
-                );
+    /**
+     * Create an unauthenticated feign client to query apis described in {@link BlockchainAdapterApiClient}.
+     * @param logLevel Feign logging level to configure.
+     * @param url Url targeted by the client.
```

Comment:
Shouldn't that `url` parameter be renamed into `baseUrl`?

Reply:
I used the same parameter name as the `target` method from feign API.
I think this is good enough.